<ui version="4.0" >
 <class>sForm</class>
 <widget class="QWidget" name="sForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>85</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>85</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>400</width>
    <height>85</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <family>Andale Mono</family>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Find (Ikke) Ændret Vare</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="leftMargin" >
    <number>6</number>
   </property>
   <property name="topMargin" >
    <number>6</number>
   </property>
   <property name="rightMargin" >
    <number>6</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_3" >
           <property name="text" >
            <string>Ændret</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="autoExclusive" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_4" >
           <property name="text" >
            <string>Ikke Ændret</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton" >
           <property name="text" >
            <string>Efter</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="autoExclusive" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2" >
           <property name="text" >
            <string>Før</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Dato</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit" >
             <property name="windowModality" >
              <enum>Qt::NonModal</enum>
             </property>
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="mouseTracking" >
              <bool>true</bool>
             </property>
             <property name="contextMenuPolicy" >
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="toolTip" >
              <string>klik på den del af dato'en du vil ændre
og brug så scroll hjulet på musen eller
knapperne til højre.</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols" >
              <enum>QAbstractSpinBox::PlusMinus</enum>
             </property>
             <property name="accelerated" >
              <bool>true</bool>
             </property>
             <property name="dateTime" >
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="minimumDate" >
              <date>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat" >
              <string>d. MMMM yyyy</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Med Vare nr:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit" />
       </item>
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>Find</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
